main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"github.com/wmaldonadoc/academy-go-q42021/infrastructure/datastore"
+	"github.com/wmaldonadoc/academy-go-q42021/infrastructure/router"
+	"github.com/wmaldonadoc/academy-go-q42021/registry"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
```

Comment:
Best practices recommend to have our imports separated into three blocks; the first block should be conformed by internal Go libraries, the second block by project's packages, and the last block by third-party packages.

Please fix this here and in all other occurrences in your project.
```suggestion
	"github.com/wmaldonadoc/academy-go-q42021/infrastructure/datastore"
	"github.com/wmaldonadoc/academy-go-q42021/infrastructure/router"
	"github.com/wmaldonadoc/academy-go-q42021/registry"
	
	"go.uber.org/zap"
	"go.uber.org/zap/zapcore"
```